api/routes/tag_routes.py

```diff
@@ -0,0 +1,57 @@
+# flake8: noqa
+from db.queries.tags.queries import insert_tags
+from db.queries.tags.queries import select_tags_for_fund_round
+from db.queries.tags.queries import select_tags_types
+from db.schemas.schemas import JoinedTagSchema
+from db.schemas.schemas import TagSchema
+from db.schemas.schemas import TagTypeSchema
+from flask import abort
+from flask import request
+
+
+def get_tags_for_fund_round(fund_id, round_id):
+
+    tags = select_tags_for_fund_round(fund_id, round_id)
+
+    if tags:
+        serialiser = JoinedTagSchema()
+        serialised_tags = [serialiser.dump(r) for r in tags]
+        return serialised_tags
+
+    abort(404, f"No tags not found for fund__round: {fund_id}__{round_id}.")
```

Comment:
Double negative here, also - wouldn't we return an empty list rather than aborting?

Seems front-end would see the 404 and get a `None` rather than en empty list, and have to handle accordingly

Reply:
```suggestion
    abort(404, f"No tags found for fund__round: {fund_id}__{round_id}.")
```